ci/stress_tests/run_application_stress_tests.sh

```diff
@@ -74,9 +74,9 @@ test_impala(){
                 s/<<<RAY_COMMIT>>>/$RAY_COMMIT/;
                 s/<<<CLUSTER_NAME>>>/$TEST_NAME/;
                 s/<<<HEAD_TYPE>>>/p3.16xlarge/;
-                s/<<<WORKER_TYPE>>>/m5.24xlarge/;
-                s/<<<MIN_WORKERS>>>/5/;
-                s/<<<MAX_WORKERS>>>/5/;
+                s/<<<WORKER_TYPE>>>/m4.16xlarge/;
+                s/<<<MIN_WORKERS>>>/9/;
+                s/<<<MAX_WORKERS>>>/9/;
```

Comment:
Why change the instance type and number of machines? Or is it a question of what your instance limits were?

Reply:
More people have m4 quota than m5 quota, which is rather rare.